biztalk/core/azure-blob-adapter.md

```diff
@@ -0,0 +1,153 @@
+ms.suite:
+ms.tgt_pltfrm:
+ms.assetid: 
+ms.custom: "biztalk-2020"
+
+---
+
+# Azure Blob storage adapter in BizTalk
+
+## Overview
+**Starting with BizTalk Server 2020**, you can send and receive messages between BizTalk Server and Azure Blob storage. 
+
+Azure Blob storage is Microsoft's object storage solution for the cloud, which is optimized for storing massive amounts of unstructured data. [What is Azure Blob storage?](https://docs.microsoft.com/azure/storage/blobs/storage-blobs-overview) provides more details.
+
+## Prerequisites
+
+* Create an [Azure blob storage account with a container](https://docs.microsoft.com/azure/storage/common/storage-create-storage-account)
+
+## Send messages to Azure Blob storage
+
+1.  In the BizTalk Server Administration console, right-click **Send Ports**, select **New**, and select **Static One-way send port**.
+
+    [Create a Send Port](../core/how-to-create-a-send-port2.md) provides some guidance.
+
+2. Enter a **Name**. In **Transport**, set the **Type** to **AzureBlobStorage**, and click **Configure** button. 
+
+3. Configure the **Azure Account** properties: 
+
+    |Use this|To do this|  
+    |---|---|  
+    | **Sign-in** | Sign into your Azure account |
+    | **Subscription** | Select the subscription that has your Azure storage account |
+    | **Resource Group** | Select your resource group that has your Azure storage account |
+
+4. Configure the **General** properties: 
+
+    |Use this|To do this|  
+    |---|---|  
+    | **Storage Authentication** | Select an authentication method. <ul><li>Typically, it's recommended to use a Shared Access Signature, which is also selected by default. You can input the Shared Access Signature connection string in the **Connection string** field to provide the authentication.</li> <li>If you are using Access keys as authentication method, a collection of storage account will be populated up in **Account** drop-down list. After you select the storage account, **Connection string** field will be automatically populated with your primary access key, which also known as **key1**. </li></ul><br />The following links are good resources to help you decide which authentication method is right for your scenario:<br/><br/>[About Azure storage accounts](https://docs.microsoft.com/azure/storage/common/storage-create-storage-account)<br/>[Using shared access signatures (SAS)](https://docs.microsoft.com/azure/storage/common/storage-dotnet-shared-access-signature-part-1) |
+    | **Blob container name** | Select the name of your Blob container from the drop-down list. The list should be auto-populated after the **Connection string** field is filled up. |
+    | **Blob name** | Name of the blob used by adapter to upload the message to Blob storage container. Macros can be used for Blob name similar to file adapter. For available Macros please refer to [Using macros in file names](restrictions-when-configuring-the-file-adapter.md#using-macros-in-file-names). |
```

Comment:
If we must mention file adapter, lets make its case as in the documentation for this adapter and provide a hyper link. Question is, if we are providing a reference to Using macros in file names, do we need to mention file adapter?

Reply:
Using macros in file names is not a standalone page, it is a section of "Restrictions when configuring the File adapter" page, so i feel it is more natual to just say this is similiar to file adapter .